wordpress-mobile/WordPress-Android
# Prepublishing Nudges : bottom sheet shown above keyboard

Fixes #12150

To test:
1. Create a new post. 
2. Click Publish before the keyboard closes. 
3. Even after it gets collapsed the first time you can press publish again.
4. Ensure the bottom sheet never look like below. 

<kbd><img src="https://user-images.githubusercontent.com/1509205/84227386-e9fba700-aaa9-11ea-96d0-c9c2ef9f8346.png" width="320"></kbd>
PR submission checklist:
